docker-compose.yml

```diff
@@ -1,27 +1,40 @@
-version: '3'
+version: '3.4'
+
+networks:
+  k6:
+  grafana:
+
 services:
   influxdb:
-    build:
-      context: .
-      dockerfile: Dockerfile.influxdb
+    image: influxdb:1.5.4
```

Comment:
Also, instead of fixed patch versions like `1.5.4` and `6.4.1`, can you leave just the major versions, so we don't have to update this file every time a new minor version is released? Not sure how reliably backwards compatible `influxdb` and `grafana` are, but can we just leave `1` and `6` as the image versions? Or at the very least, `1.5` and `6.4`?

Reply:
Hmm actually, why are we using InfluxDB 1.5, when 1.7.8 seems to be the latest stable version?

Reply:
Ah, `1.5` was on top of the list so I assumed that was the latest version :stuck_out_tongue: 

Unfortunately Grafana does not support tags for only the majors https://github.com/grafana/grafana/issues/16193. I will change influxDB to `influxdb:1.7-alpine`, but would you prefer `latest` or a fixed patch for Grafana?

Reply:
Personally, I'd probably prefer `latest`, so we don't have to update it every so often. Looking at the grafana docker tags, `latest` is the latest stable release, and they have a separate `master` tag like us, so it should be stable enough...

Reply:
@na-- It would be safer to have them version pinned (as everything else) if we want to ensure a stable environment. I wouldn't mind occasional version bump commits that are pre-approved by our builds.

Reply:
Safer, yes, which would be very important if this was in some way an official k6 release. But I'm not sure that's the expectation from this docker-compose file, given that it actually builds the latest `master` k6... I think having the `latest` grafana (and probably `latest` k6 image, instead of building it) wouldn't be a huge issue, and I'd argue that it's probably better for us to know if there are any problems.

Reply:
Ah, yes, I missed the detail that it's already building k6 from `master`.

Fair enough, if the expectation is already to use the latest (stable) k6, making the other images `latest` would align with that.